### PR TITLE
Adding details about Boolean Data types and FROM subitems

### PR DESCRIPTION
**Pull request by Rodrigossz**

### CODE REVIEW

**Review by southpolesteve — DISMISSED** [review]

File: sdk/cosmos/azure-cosmos/README.md [hunk 1]
```diff
@@ -106,6 +106,23 @@ As of August 2020 the features below are **not supported**.
 
 If you want to use Python SDK to perform bulk inserts to Cosmos DB, the best alternative is to use [stored procedures](https://docs.microsoft.com/azure/cosmos-db/how-to-write-stored-procedures-triggers-udfs) to write multiple items with the same partition key.
 
+## Boolean Data Type
+
+While the Python language [uses](https://docs.python.org/3/library/stdtypes.html?highlight=boolean#truth-value-testing) "True" and "False" for boolean types, Cosmos DB [accepts](https://docs.microsoft.com/azure/cosmos-db/sql-query-is-bool) "true" and "false" only. In other words, the Python language uses Boolean values with the first uppercase letter and all other lowercase letters, while Cosmos DB and its SQL language use only lowercase letters for those same Boolean values.How to deal with this challenge?
```
Comment (zfoster) [inline sdk/cosmos/azure-cosmos/README.md:111]:
```suggestion
While the Python language [uses](https://docs.python.org/3/library/stdtypes.html?highlight=boolean#truth-value-testing) "True" and "False" for boolean types, Cosmos DB [accepts](https://docs.microsoft.com/azure/cosmos-db/sql-query-is-bool) "true" and "false" only. In other words, the Python language uses Boolean values with the first uppercase letter and all other lowercase letters, while Cosmos DB and its SQL language use only lowercase letters for those same Boolean values. How to deal with this challenge?
```

File: sdk/cosmos/azure-cosmos/README.md [hunk 2]
```diff
@@ -106,6 +106,23 @@ As of August 2020 the features below are **not supported**.
...
+
+* Your JSON documents created with Python must use "True" and "False", to pass the language validation. The SDK will convert it to "true" and "false" for you. Meaning that "true" and "false" is what will be stored in Cosmos DB. 
+* If you retrieve those documents with the Cosmos DB Portal's Data Explorer, you will see "true" and "false". 
+* If you retrieve those documents this Python SDK, "true" and "false" values will be automatically converted to "True" and "False".
```
Comment (zfoster) [inline sdk/cosmos/azure-cosmos/README.md:115]:
```suggestion
* If you retrieve those documents with this Python SDK, "true" and "false" values will be automatically converted to "True" and "False".
```

File: sdk/cosmos/azure-cosmos/README.md [hunk 3]
```diff
@@ -106,6 +106,23 @@ As of August 2020 the features below are **not supported**.
...
+
+## SQL Queries x FROM Clause Subitems
+
+This SDK uses the [query_items](https://docs.microsoft.com/python/api/azure-cosmos/azure.cosmos.containerproxy?preserve-view=true&view=azure-python#query-items-query--parameters-none--partition-key-none--enable-cross-partition-query-none--max-item-count-none--enable-scan-in-query-none--populate-query-metrics-none----kwargs-) method to submit SQL queries to Azure Cosmos DB. 
+
+Cosmos DB SQL language allows you to [get subitems by using the FROM clause](https://docs.microsoft.com/azure/cosmos-db/sql-query-from#get-subitems-by-using-the-from-clause), to reduce the source to a smaller subset. As an example, you can use `select * from Families.children` instead of `select * from Families`. But please note that:
+
+* For SQL queries using the `query_items` method, this SDK demands that you specify the `partition_key` or use the `enable_cross_partition_query` flag. 
+* If you are getting subitems and specifing the `partition_key`, please make sure that your partition key is included in the subitems, what is not true for most of the cases.
```
Comment (zfoster) [inline sdk/cosmos/azure-cosmos/README.md:124]:
```suggestion
* If you are getting subitems and specifying the `partition_key`, please make sure that your partition key is included in the subitems, which is not true for most of the cases.
```

**Review by zfoster — DISMISSED** [review]
added a couple little things to change

**Comment by Rodrigossz** [comment]
> added a couple little things to change

Thank you @zfoster

**Review by zfoster — APPROVED** [review]

**Pull request merged**